.github/workflows/build-ci.yml

```diff
@@ -48,31 +46,19 @@ jobs:
             -   uses: "actions/checkout@v4"
 
             -   name: "Create MongoDB Replica Set"
-                if: ${{ matrix.mongodb != 'Atlas' }}
                 run: |
                     docker run --name mongodb -p 27017:27017 -e MONGO_INITDB_DATABASE=unittest --detach mongo:${{ matrix.mongodb }} mongod --replSet rs --setParameter transactionLifetimeLimitSeconds=5
 
                     if [ "${{ matrix.mongodb }}" = "4.4" ]; then MONGOSH_BIN="mongo"; else MONGOSH_BIN="mongosh"; fi
-                    until docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "db.runCommand({ ping: 1 })"; do
-                      sleep 1
-                    done
-                    sudo docker exec --tty mongodb $MONGOSH_BIN 127.0.0.1:27017 --eval "rs.initiate({\"_id\":\"rs\",\"members\":[{\"_id\":0,\"host\":\"127.0.0.1:27017\" }]})"
-
-            -   name: "Create MongoDB Atlas Local"
-                if: ${{ matrix.mongodb == 'Atlas' }}
-                run: |
-                    docker run --name mongodb -p 27017:27017 --detach mongodb/mongodb-atlas-local:latest
-                    until docker exec --tty mongodb mongosh 127.0.0.1:27017 --eval "db.runCommand({ ping: 1 })"; do
-                      sleep 1
-                    done
-                    until docker exec --tty mongodb mongosh 127.0.0.1:27017 --eval "db.createCollection('connection_test') && db.getCollection('connection_test').createSearchIndex({mappings:{dynamic: true}})"; do
+                    until docker exec --tty mongodb $MONGOSH_BIN --eval "db.runCommand({ ping: 1 })"; do
```

Comment:
Host removed, related to https://github.com/mongodb/laravel-mongodb/pull/3205#discussion_r1915520686